packages/block-editor/src/store/selectors.js

```diff
@@ -2658,8 +2658,11 @@ export function wasBlockJustInserted( state, clientId, source ) {
  * @param {Object} state Global application state.
  * @return {string|undefined} Client Id of the last inserted block.
  */
-export function getLastInsertedBlockClientId( state ) {
-	return state?.lastBlockInserted?.clientId;
+export function __experimentalGetLastInsertedBlockClientId( state ) {
+	return (
+		state?.lastBlockInserted?.clientIds?.length &&
+		state?.lastBlockInserted?.clientIds[ 0 ]
```

Comment:
We _could_ update this selector to return _all_ the clientIds and allow the user to deal with which one they want to count as the "last" or "latest".

So if there is a single block you'd then consume it thus

```
const lastInsertedBlockClientId = __experimentalGetLastInsertedBlockClientIds()[0]
```

Reply:
I think we should return the array.

Reply:
Agreed, picking the first one is very arbitrary

Reply:
Excellent. We have a consensus. So I assume we:

- rename the selector to `__experimentalGetLastInsertedBlockClientIds` (plural)
- deprecate the original selector

Reply:
I'm going to follow the guidance and advice in the docs

https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/backward-compatibility.md#how-to-preserve-backward-compatibility-for-a-javascript-function

Reply:
I vote we keep it experimental for now.